apps/wallet-mobile/src/TxHistory/TxHistory.tsx

```diff
@@ -36,7 +37,10 @@ export const TxHistory = () => {
     setActiveTab(tab)
   }
 
-  const {sync, isLoading} = useSync(wallet)
+  const {sync, isLoading: isLoadingWallet} = useSync(wallet)
+  const {isLoading: isLoadingPoolTransition} = usePoolTransitionModal()
```

Comment:
since is requested while opening the wallet, it should swallow errors, otherwise a failed request won't allow the user to open the wallet, it will happen again when opening so it looks safe.

Reply:
it can also leverage the error boundary with a retry

Reply:
Not sure I understand. If usePoolTransitionModal got an error from usePoolTransition, isLoading should still go to false.

Reply:
what will happen to user if getPoolTrasitino fails?

Reply:
It's inside useQuery, so you'll get an error returned there. Since it's not used, then nothing out of the ordinary happens.